feat: Adapt the document to the release and add JSON views doc (#127)
lenra-io/docs: src/markdown/features/data-management.md

There is two ways to handle data in Lenra:
- in the listeners: you can use the [API](#api) to manage your data
- in the views: you can use the [View component](../references/components-api/components/view.html) to adapt your application interface

## @me

The `@me` value is a special string that represents the current user.
It can be used for document manipulation and queries.

> To use it, you need just need to set the `@me` value to a field of the document like is done in our templates (see [the create a tempalte from scratch guide](../guides/create-from-scratch.html) to better understand how to use it).

Reviewer: ```suggestion
To use it, you just need to set the `@me` value to a field of the document as done in our templates (see [the create a template from scratch guide](../guides/create-from-scratch.html) to better understand how to use it).
```
